### PR TITLE
Allow string label in community `construct`

### PR DESCRIPTION
**Pull request by adamnsch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,17 +89,21 @@ To do this we provide the `gds.alpha.graph.construct` method with node data fram
...
+        "otherProperty": [0.1, 0.2, 0.3, 0.4]
+    }
 )
 
 relationships = pandas.DataFrame(
-    "sourceId": [0, 1, 2, 3],
-    "targetId": [1, 2, 3, 0],
-    "relationshipType": ["REL", "REL", "REL", "REL"],
-    "weight": [0.0, 0.0, 0.1, 42.0]
+    {
```
Comment (FlorentinD) [inline README.md:101]: is there a difference to use the curly brackets here?

File: graphdatascience/query_runner/cypher_graph_constructor.py [hunk 2]
```diff
@@ -59,23 +62,32 @@ def _is_enterprise(self) -> bool:
...
         if "labels" in node_df.keys():
             label_index = node_df.columns.get_loc("labels")
             label_query = f", node[{label_index}] as labels"
 
+            # Make sure every node has a list of labels
+            for node in node_list:
+                labels = node[label_index]
+                if isinstance(labels, List):
+                    continue
+                node[label_index] = [labels]
```
Comment (FlorentinD) [inline graphdatascience/query_runner/cypher_graph_constructor.py:79]: so now we pack strings into a list :)

**Review by FlorentinD — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (adamnsch) [reply README.md:101]: It was wrong before. The `DataFrame` constructor takes a dictionary which now it it :)

**Pull request merged**